dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -170,6 +170,12 @@ log.access.max.file.size.desc=Maximum size of a log file, in megabytes, before b
 log.access.max.files=Maximum log files
 log.access.max.files.desc=Maximum number of log files that will be kept, before the oldest file is removed; a value of 0 means no limit.
 
+log.access.rollover.start.time=Rollover start time for time based log rollover
+log.access.rollover.start.time.desc=The scheduled time of day when logs should first rollover. The rollover interval duration will begin at rollover start time. Valid values follow a 24-hour ISO-8601 datetime format of HH:MM, where 00:00 represents midnight. Padding zeroes are required. If rolloverInterval is specified, the default value of rolloverStartTime is 00:00, midnight. If both rolloverInterval and rolloverStartTime are unspecified, time based log rollover is disabled. Note: rolloverStartTime does not apply to the console.log file.
```

Comment:
Suggested changes to remove "should" and "will," to change "rollover" (noun) to "roll over" (verb), and to use "zeros" not "zeroes":

The scheduled time of day for logs to first roll over. The rollover interval duration begins at rollover start time. Valid values follow a 24-hour ISO-8601 datetime format of HH:MM, where 00:00 represents midnight. Padding zeros are required.